apps/src/code-studio/components/ExportDialog/Dialog.jsx

```diff
@@ -692,7 +692,7 @@ class ExportDialog extends React.Component {
           className="modal-content no-modal-icon"
           style={{position: 'relative'}}
         >
-          <p className="dialog-title">Export your project</p>
+          <p className="dialog-title">Export your project (Beta)</p>
```

Comment:
Can we i18n this? It looks like almost all of this dialog is already 18n-ready, except for a few buttons and titles.

Reply:
To my knowledge, we have no plans for applab and gamelab to be translated, so features like this that are only for those 2 apps have not had it added. We can make the change here, but all of the pages under `components/ExportDialog` are also English only, so there isn't much point.

The reason you see other `i18n` strings in this dialog is that it has some similar messages/warnings as the Share dialog, so I lifted those (as translated strings). The Share dialog is used outside of applab and gamelab, so it is 100% i18n.

Reply:
In general, we try to avoid hardcoding strings as it makes future string management more difficult. Putting it in the i18n file helps keep us flexible.

Reply:
@jmkulwik @epeach 
My understanding is that we don't put applab and gamelab text in i18n, so that we don't confuse translators by having them work on translations that aren't planned or needed. Has this changed?
The only places where i18n is used here are for existing strings that were already referenced with the same text. Nothing that is specific to applab/gamelab has been translated throughout the Code Studio codebase, unless this just started recently.

Reply:
My understanding is we use i18n for more than just translations. We also use it for string management. Also, although we aren't spending resources on translating CSP, that doesn't mean we want to disallow internationalizing Applab/Gamelab. Adding @Hamms to comment on this as he has a fuller understanding of our internationalization+i18n strategy.

Reply:
We do use i18n primarily for translations, and we do prefer to only include strings that we actually want to be translated to avoid creating noise for our translators. However, we do have plans to internationalize applab and gamelab in the not-to-distant future, so in this case I would love to see some internationalizable strings in this component!